Helper/Data.php

```diff
@@ -80,7 +80,7 @@ class Data extends AbstractHelper
         ...self::NATIVE_SENTRY_CONFIG_KEYS,
         'logrocket_key'                       => ['type' => 'string'],
         'log_level'                           => ['type' => 'int'],
-        'logger_log_level'                    => ['type' => 'int', 'default' => \Monolog\Logger::WARNING],
+        'logger_log_level'                    => ['type' => 'int', 'default' => \Monolog\Level::Warning],
```

Comment:
Unfortunately this class is not available in monolog 2.x, used for Magento <2.4.8
We're not prepared to drop support for the lower versions yet, for now i think
```suggestion
        'logger_log_level'                    => ['type' => 'int', 'default' => 300 /* \Monolog\Level::Warning */],
```
will suffice